actors/builtin/miner/policy.go

```diff
@@ -30,7 +30,7 @@ const PoStLookback = abi.ChainEpoch(1) // PARAM_FINISH
 const ElectionLookback = PoStLookback // PARAM_FINISH
 
 // Number of sectors to be sampled as part of windowed PoSt
-const NumWindowedPoStSectors = 200 // PARAM_FINISH
+const NumWindowedPoStSectors = 1 // PARAM_FINISH
```

Comment:
I'm worried this will mask issues until we push it back up, maybe with little time to resolve whatever they are. What does this simplify?

Reply:
The post we are running now cares about this, The post we will ship with testnet will not.